### PR TITLE
Update /financial-services page design

### LINKED ISSUE #9221: [UX fix] /financial-services strip is horrible
Can someone look at this strip and propose a better solution to adding the whitepaper?

image

[copy doc](https://docs.google.com/document/d/1SEd2nV0GplBTG4lZ9RuhjNXrphU22nGP4Fpgd5g_9BQ/edit?ts=601c07bc#heading=h.7y7b3q239v01)

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Update hero, 'Drive business agility' and 'Software strategy acceleration' sections

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/financial-services
- Check page against new [design](https://user-images.githubusercontent.com/36884067/109687240-f5808800-7b7a-11eb-88c2-39032f7be79b.jpg)

## Issue / Card

Fixes #9221 

## Screenshots

image

image

image

### CODE REVIEW

**Comment by anasereijo** [comment]
this is looking good! just some small things:

- [ ]  hide the logos from "Drive business agility" on mobile version
- [ ] show the whitepaper section on mobile
- [ ] make "Trusted by executives. Loved by leaders." an h4
- [ ] show canonical and ubuntu logos on mobile otherwise the text doesn't make sense
- [ ] check any copy changes in the copydoc

Thanks!

**Comment by bethcollins92** [comment]
Thanks @anasereijo, made those changes

**Comment by anasereijo** [comment]
great, thanks! 👍

File: templates/financial-services/index.html [hunk 1]
```diff
@@ -22,7 +22,7 @@ <h1>Finserv open source solutions</h1>
   </div>
 </section>
 
-<section class="p-strip is-bordered">
+<section class="p-strip--light">
   <div class="row">
     <h2 class="p-muted-heading u-align--center">INNOVATING ON UBUNTU WITH CANONICAL</h2>
```
Comment (solazio) [inline templates/financial-services/index.html:27]: This can be in lower case.

File: templates/financial-services/index.html [hunk 2]
```diff
@@ -92,67 +92,67 @@ <h2 class="p-muted-heading u-align--center">INNOVATING ON UBUNTU WITH CANONICAL<
...
+          height="41",
           hi_def=True,
           loading="lazy"
           ) | safe
           }}
         </li>
-      </ul>
-      <ul class="p-inline-images u-no-margin">
-        <li class="p-inline-images__item">
+        <li class="p-inline-images__item" style="margin-top: 0.75rem;">
```
Comment (solazio) [inline templates/financial-services/index.html:111]: This hack is not great. I think you should do it with `display: flex` and `align-items: baseline` or similar.
image

**Review by solazio — COMMENTED** [review]
A few small comments in the code, but the main problem is that the page has no semantic structure for the headings.
We should amend that and have one `<h1>` and then `<h2>`, etc.

image

**Comment by bethcollins92** [comment]
Thanks @solazio, made suggested changes and ready for re-review

**Review by solazio — APPROVED** [review]
Looks good :+1:. Thanks

**Pull request merged**